commands/commands.go

```diff
@@ -210,30 +210,6 @@ type ErrorWithStack interface {
 	Stack() []byte
 }
 
-// determineIncludeExcludePaths is a common function to take the string arguments
-// for include/exclude and derive slices either from these options or from the
-// common global config
-func determineIncludeExcludePaths(includeArg, excludeArg string) (include, exclude []string) {
```

Comment:
I actually think this function was better where it was; it's specifically about converting string arguments to slices or defaulting and so belongs firmly unexported in `command` in my view. I don't see any benefit in promoting it to an exported function in `Configuration`, doing so widens that scope outside its remit IMO (which I think you were feeling too based on your follow up comment).